docs/developing.asciidoc

```diff
@@ -0,0 +1,80 @@
+// begin header
+// end header
+
+toc::[]
+
+== Setting up Certman-operator for local dev
+=== Requirements
+To develop certman you will need the following:
+
+. Access to the hive repo
```

Comment:
Is this the `github.com/openshift/hive` repo? Maybe add a link or hyperlink for `hive` here